README.md

```diff
@@ -21,19 +29,28 @@ Edit the `download.sh` script with the signed url provided in the email to downl
 
 ## Inference
 
-The provided `example.py` can be run on a single or multi-gpu node with `torchrun` and will output completions for two pre-defined prompts. Using `TARGET_FOLDER` as defined in `download.sh`:
+The provided `example_xla.py` can be run on a TPU VM with `gcloud compute tpus tpu-vm ssh` and will output completions for one pre-defined prompts. Using `TARGET_FOLDER` as defined in `download.sh`:
 ```
-torchrun --nproc_per_node MP example.py --ckpt_dir $TARGET_FOLDER/model_size --tokenizer_path $TARGET_FOLDER/tokenizer.model
+gcloud compute tpus tpu-vm ssh ${TPU_NAME} --project ${PROJECT} --zone ${ZONE} --worker=all --command='
+export PJRT_DEVICE=TPU
+export TOKENIZER_PATH=$TARGET_FOLDER/tokenizer.model
+export CKPT_DIR=$TARGET_FOLDER/model_size
+
```

Comment:
Hmm Should we also include the torch_xla wheel here?

Reply:
Yes, probably should be in the `##setup` above.